importer/markdown/src/test/java/org/itsallcode/openfasttrace/importer/markdown/ITMarkdownImporter.java

```diff
@@ -246,13 +248,49 @@ void testItemIdSupportsUmlauts()
         final List<SpecificationItem> items = runImporterOnText(
                 "`### Die Implementierung muss den Zustand einzelner Zellen ändern.\n"
                         + "`req~zellzustandsänderung~1`\n"
-                        + "Diese Anforderung ermöglicht die Aktualisierung des Zustands von lebenden und toten Zellen in jeder Generation.\n"
+                        + "Diese Anforderung ermöglicht die Aktualisierung des Zustands von lebenden und toten Zellen"
+                        +" in jeder Generation.\n"
                         + "Needs: arch");
         assertThat(items, AutoMatcher.contains(SpecificationItem.builder()
                 .id(SpecificationItemId.createId("req", "zellzustandsänderung", 1))
                 .description(
-                        "Diese Anforderung ermöglicht die Aktualisierung des Zustands von lebenden und toten Zellen in jeder Generation.")
+                        "Diese Anforderung ermöglicht die Aktualisierung des Zustands von lebenden und toten Zellen"
+                                +  " in jeder Generation.")
                 .location("file name", 2).addNeedsArtifactType("arch")
                 .build()));
     }
+
+    @Test
+    void testRecognizeItemTitleWithUnderlines()
```

Comment:
Make this a parameterized test with other options for underlining:
* Only three `---` chars
* The alternative `===`
* More than three chars but less than the heading length
* Longer more chars than the heading length

Add another test with less than three chars, check that this is not recognized as a heading.